api/searchctrl.md

```diff
@@ -2,29 +2,40 @@
 
 REST API located at /api/searchctrl
 
-The searchctrl group is used to query information about active searches and optionally invoke some action.  We currently provide the ability to query all active searches, get information about a specific search, background a an active search, archive an active search, and delete/terminate a search.
+The searchctrl group is used to query information about persistent searches and optionally invoke some action.  We currently provide the ability to query all active searches, get information about a specific search, background a an active search, archive an active search, and delete/terminate a search.
 
 ## Basic API overview
 
-The basic action here is to perform a GET, DELETE, or PATCH on a REST url
+The basic action here is to perform a GET, DELETE, or PATCH on a REST url.
+
+## Search States
+
+Searches can be in any of the following states:
+
+- Active: The search is running and/or finished and there is a session attached to it.
+- Backgrounded: The search is actively running but marked in a way that it will persist without an attached session.
+- Saving: The search has been marked as saved, but is still waiting for completion to move its contents to a persistent location.
+- Saved: The search is marked as saved and moved to the appropriate persistent location.
+- Dormant: The search is being kept (background or saved) and no sessions are attached.
+- Attached: The search is saved and a session has re-attached to it.
```

Comment:
These are not exclusive states, something can be ACTIVE/BACKGROUNDED or SAVED/ATTACHED, or DORMANT/SAVED.

I would suggest adding a blurb explaining this and noting that the states may be blended.